requirements.txt

```diff
@@ -1,4 +1,6 @@
 -e git+http://github.com/xamox/pygame
 MySQL-python==1.2.5
```

Comment:
Since we're using sqlite, we shouldn't require MySQL-python and in fact the installation of this will fail on some machines that don't have MySQL installed.